cloud-mock/src/main/java/com/sequenceiq/cloudbreak/cloud/mock/MockInstanceConnector.java

```diff
@@ -37,6 +37,14 @@
             CloudVmInstanceStatus instanceStatus = new CloudVmInstanceStatus(instance, InstanceStatus.CREATED);
             cloudVmInstanceStatuses.add(instanceStatus);
         }
+
+        MockCredentialView mockCredentialView = mockCredentialViewFactory.createCredetialView(authenticatedContext.getCloudCredential());
+        LOGGER.info("stop instance statuses to mock spi, server address: " + mockCredentialView.getMockEndpoint());
```

Comment:
"stop instance statuses". maybe should be "start instance statuses"?

Reply:
yepp